### ISSUE TITLE
Installer cannot detect disk

### ISSUE THREAD

**Issue by sf736** [dosdude1/macos-catalina-patcher#55]
MacBook7,1
Samsung SSD 860 QVO 1TB

Issue:  Patched Catalina Installer cannot detect SSD
Work around: 
- Boot Linux, SSD is found
- Erase SSD.  Create new GPT table
- Create a single partition for the whole disk
- Boot Installer again, Disk is found.
- Erase and Install is more or less OK.
- When reboot, system might show the greyed "Stop" sign.  Just force power off, and power on again, then it can boot.

I am not sure why this is happening.  But I will put in an old HDD to cross check, if it is related to this Samsung SSD 860 QVO or not.

**Comment by TonyVee73** [comment]
This happened to me the 1st time I tried to install the Catalina Patcher on my Macbook Pro 6,2. I had 16gb ram and SanDisk Ultra 3D NAND 1TB Internal SSD that worked fine with High Sierra. 

After a bunch of trouble shooting, it ended up detecting hard drives (SSD and HDD) when I removed the 16GB ram and replaced it with the original 4GB  I had. I've since replaced the 4GB with 8GB and it's running perfect.

**Comment by sf736** [comment]
Yes, I read from the Google searches that some are having bad Rams and cables, replaced and all is fine.

For me, I don't think it is hardware related.

As when I boot Windows installer, or Linux LiveCD, the drive is always here.

If I created certain partitions in the Drive, Catalina Patcher always detects the drive.

Thus I guess Catalina is being smart trying to hide any device that it want to hide.

Even when I have a running patched Catalina in the SSD, and when I use the Patcher USB to enter Recovery, it still can report not internal drive.

**Comment by priyank93** [comment]
Hi, I would like to know how can I resolve this issue. Everytime I try to boot from USB to install Catalina using the patcher tool, disk utilities won't detect any drives for me to install the OS.

**Comment by r8cole** [comment]
I had the same issue. Would not detect the internal drive. Tried restarting installed. Then I read about running disk utilities to check the internal drive. Which I did. I read about converting the drive to APFS using disk doctor. So I did held down Command R and started in Recovery mode. After checking the disk with the check-up utility. I unmounted the drive in hopes it would allow me to convert to APFS. It did not. So I tried one last time rebooting Catalina Patcher from the thumb drive. This time the internal drive showed up. It may have been the unmounting option I did on the internal, as I did not remount before I restarted. Who knows. It's converting the internal drive now. Fingers crossed.
